tools/src/main/scala/scala/scalanative/build/LLVM.scala

```diff
@@ -52,8 +52,9 @@ private[scalanative] object LLVM {
           if (config.targetsWindows) Seq("-g")
           else Nil
         }
-        val flags = opt(config) +: stdflag ++: platformFlags ++: "-fvisibility=hidden" +:
-          config.compileOptions
+        val flags =
+          opt(config) +: stdflag ++: platformFlags ++: "-fvisibility=hidden" +:
```

Comment:
You could probably avoid the extra line here by wrapping the "-fvisibility=hidden" +: onto the next line so the two lines are more equal and the first line won't wrap at 80.